--- a/kernel/sched/fair.c.orig	2023-08-31 17:59:48.860595371 +0900
+++ a/kernel/sched/fair.c	2023-08-31 17:59:58.377604497 +0900
@@ -19,9 +19,6 @@
  *
  *  Adaptive scheduling granularity, math enhancements by Peter Zijlstra
  *  Copyright (C) 2007 Red Hat, Inc., Peter Zijlstra
- *
- *  Remove energy efficiency functions by Alexandre Frade
- *  (C) 2021 Alexandre Frade <kernel@xanmod.org>
  */
 #include <linux/energy_model.h>
 #include <linux/mmap_lock.h>
