--- a/kernel/sched/fair.c.orig	2023-10-26 17:05:05.304131064 +0900
+++ b/kernel/sched/fair.c	2023-10-26 17:07:04.681605814 +0900
@@ -673,7 +673,7 @@
 	return __node_2_se(next);
 }
 
-#ifdef CONFIG_SCHED_DEBUG
+/*#ifdef CONFIG_SCHED_DEBUG*/
 struct sched_entity *__pick_last_entity(struct cfs_rq *cfs_rq)
 {
 	struct rb_node *last = rb_last(&cfs_rq->tasks_timeline.rb_root);
@@ -704,7 +704,7 @@
 
 	return 0;
 }
-#endif
+/*#endif*/
 
 /*
  * delta /= w
