--- PKGBUILD.orig	2022-11-08 22:42:07.235480846 +0900
+++ PKGBUILD	2022-11-08 22:43:32.287435866 +0900
@@ -121,7 +121,9 @@ prepare() {
 
   # Apply Xanmod patch
   patch -Np1 -i ../patch-${pkgver}-xanmod${xanmod}
-
+  patch -Np1 -i ../../bore.patch
+  patch -Np1 -i ../../O3.patch
+  
   msg2 "Setting version..."
   scripts/setlocalversion --save-scmversion
   echo "-$pkgrel" > localversion.10-pkgrel
@@ -145,6 +147,9 @@ prepare() {
     scripts/config --enable LTO_CLANG_THIN
   fi
 
+  scripts/config --disable CONFIG_HZ_500
+  scripts/config --enable CONFIG_HZ_1000
+  
   # CONFIG_STACK_VALIDATION gives better stack traces. Also is enabled in all official kernel packages by Archlinux team
   scripts/config --enable CONFIG_STACK_VALIDATION
 
